config/samples/apim_v1alpha1_ratelimitpolicy.yaml

```diff
@@ -1,30 +1,44 @@
+---
 apiVersion: apim.kuadrant.io/v1alpha1
 kind: RateLimitPolicy
 metadata:
-  name: ratelimitpolicy-sample
-  namespace: kuadrant-system
-  labels:
-    app: toystore
+  name: toystore
 spec:
-  networkingRef:
-    - type: VirtualService
-      name: toystore
   routes:
     - name: get-toy
-      stage: BOTH
-      actions:
-        - generic_key:
-            descriptor_key: get
-            descriptor_value: "yes"
+      rateLimits:
+        - stage: BOTH
+          actions:
+            - generic_key:
+                descriptor_key: get-toy
+                descriptor_value: "yes"
+            - generic_key:
+                descriptor_key: other-get-toy
+                descriptor_value: "yes"
     - name: add-toy
-      stage: POSTAUTH
+      rateLimits:
+        - stage: POSTAUTH
+          actions:
+            - generic_key:
+                descriptor_key: add-toy
+                descriptor_value: "yes"
+    - name: delete-toy
+      rateLimits:
+        - stage: PREAUTH
+          actions:
+            - generic_key:
+                descriptor_key: delete-toy
+                descriptor_value: "yes"
+  rateLimits:
+    - stage: PREAUTH
       actions:
         - generic_key:
-            descriptor_key: admin
+            descriptor_key: vhaction
             descriptor_value: "yes"
+
   limits:
```

Comment:
do you think with this sample it is worth adding a set of limits for the vhost level?

Reply:
Sure, I can add one for the vhost level as well.

Reply:
Added